src/styles/utils/variables.scss

```diff
@@ -0,0 +1,21 @@
+$color-dark: #060b35;
+$color-light: #616070;
+$color-casual: #fff;
+$color-hover: #00acdc;
+$color-card-border: #f3f3f3;
+
+$card-width: 200px;
+$card-margin-right: 48px;
+$catalog-padding: 40px;
+
+$catalog-width: 4 * $card-width + 3 * $card-margin-right + 2 * $catalog-padding;
```

Comment:
Its better to move amount of the card to the separate variable, so you could easily change it later on

Reply:
done